site/docs/spec.md

```diff
@@ -254,6 +254,22 @@ Notes:
 2. The width, `W`, used to truncate decimal values is applied using the scale of the decimal column to avoid additional (and potentially conflicting) parameters.
 
 
+### Sorting
+
+Users could sort their data within partitions by columns to gain performance. The information on how the data is sorted could be declared per data or delete file, by a **sort order**.
```

Comment:
Could you change both uses of "could" to "can" in this paragraph? That way there is no implied past tense.